README.md

```diff
@@ -96,10 +96,10 @@ contain 3rd party code as documented in `NOTICE.txt`.
 # Trademarks
 
 This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft
-trademarks or logos is subject to and must follow
+trademarks or logos are subject to and must follow

```

Comment:
The target is singular "use" so it was correct before. Also this statement came from lawyers and I don't think we should mess with it.